doc/flakes/design.md

```diff
@@ -2,92 +2,80 @@
 
 ## Goals
 
-* To provide Nix repositories with an easy and standard way to
-  reference other Nix repositories.
+* Standard and easy dependency handling in Nix
 
-* To allow such references to be queried and updated automatically.
+* Discoverability: Be able to query and update dependencies automatically
```

Comment:
Idem: this suggests we're querying/updating package dependencies (like the many auto-updates that exist for Nixpkgs).